mltrace/client.py

```diff
@@ -100,6 +100,7 @@ def log_component_run(
         component_run_sql.add_notes(component_run_dict["notes"])
 
     component_run_sql.set_git_hash(component_run_dict["git_hash"])
+    component_run_sql.set_git_tags(component_run_dict['git_tags'])
```

Comment:
s/`'git_tags'`/`"git_tags"` -- sorry i know this is really picky but consistency!

Reply:
that's observant!